src/test/client.js

```diff
@@ -338,6 +399,13 @@ function simulateConnection(dumpPath, resultPath, ua, protocolV) {
 function runTest() {
```

Comment:
A test that would somehow simulate  a server restart would be cool, but I assume that would be a bit convoluted, maybe an isolated test for the OprhanFileHelper? wdyt?